Document/Poll.php

```diff
@@ -28,13 +29,17 @@ class Poll extends Content
     protected $title;
 
     /**
-     * @var string
+     * @var array
```

Comment:
Can you add `Option[]` as type-hint? Because this attribute can only store the class `Option`.